packages/image_picker/image_picker_ios/ios/image_picker_ios/Sources/image_picker_ios/FLTImagePickerPlugin.m

```diff
@@ -99,6 +99,7 @@ - (void)launchPHPickerWithContext:(nonnull FLTImagePickerMethodCallContext *)con
   PHPickerConfiguration *config =
       [[PHPickerConfiguration alloc] initWithPhotoLibrary:PHPhotoLibrary.sharedPhotoLibrary];
   config.selectionLimit = context.maxItemCount;
+  config.preferredAssetRepresentationMode = PHPickerConfigurationAssetRepresentationModeCurrent;
```

Comment:
nice! out of curiosity does this get used in file selector too? I remember that this bug only happened on image picker and not file selector.

Reply:
yeah the bug doesnt seem to affect file selector. Filw selector uses `UIDocumentPickerViewController` and it gives access to the file directly